CONTRIBUTING.md

```diff
@@ -66,27 +66,46 @@ If this raises any issues, try automatically fixing them
 make format
 ```
 
-# Releasing
+# Updating examples
 
-Releases of this repository lag releases of Saturn Cloud. Anytime after a new release makes it to [hosted](https://app.community.saturnenterprise.io) and you want to take advantage of some new features in it, you can make a release of this repo.
+The goal is that the you should be able to push changes to main and see those changes as quickly as possible on Saturn without breaking anything that already exists.
 
-Follow these steps:
+To support that goal, there should be only one branch of this repo for every recipe schema version.
 
-1. Create a pull request into `main` which updates all of the `"image_uri"` entries in `saturn.json` files to the latest images released from https://github.com/saturncloud/images (if you want).
-2. Create a new branch from `main`. It's name must start with `release-`.
+## Use new versions of images (common)
 
-    ```shell
-    git pull origin main
-    git checkout -b release-2022.01.06
-    git push release-2022.01.06
-    ```
+After new images are created:
 
-3. On `main` update RECIPE_SCHEMA_VERSION to the version on [hosted](https://app.community.saturnenterprise.io).
-4. Update all the `schema_version` fields in all the recipes and make any changes.
-5. Update the button on README.md to point to the new RECIPE_SCHEMA_VERSION
-6. Create a pull request with these changes with `main` as the base.
+1. Create a pull request into `main` which updates all of the `"image_uri"` entries in `saturn.json` files to the latest images released from https://github.com/saturncloud/images.
+2. Run integration tests using that branch.
+3. If the images are not yet available on one of the staging instances, ask someone on the devops team to update the images there.
+4. Manually test any examples that you are worried about.
+5. Once you are satisfied, ask someone on the devops team to update the images on hosted.
+6. Merge the pull request.
```

Comment:
This part can't work without further modifications, like we discussed yesterday - the examples _are_ how the images get updated.

```suggestion
3. Merge the pull request.
4. If the images are not yet available on one of the staging instances, ask someone on the devops team to update the examples there.
5. Once you are satisfied, ask someone on the devops team to update the examples on hosted.
```

This modification would work with what we have now.

Reply:
yes sorry I should have clarified what I am trying to write in this document. I am trying to write about an ideal future state where we _do have_ separate "make image-examples" "make examples". There are two reasons for this. One is that there are examples that exist in this repo that aren't part of the default resource templates that we call "examples" in saturn. Those examples might depend on images that are different than the default resource templates. So I think we need some way of saying "these are all the images that any example will need" on hosted. And we need a way to update those images.